src/BulkDocs/BulkDocs.php

```diff
@@ -85,6 +86,12 @@ public function save() {
     foreach ($this->entities as $entity) {
       try {
         $entity->_rev->new_edit = $this->newEdits;
+
+        // This will save stub entities in case the entity has entity reference
+        // fields and a referenced entity does not exist or will update stub
+        // entities with the correct values.
+        \Drupal::service('relaxed.stub_entity_processor')->processEntity($entity);
```

Comment:
Let's inject this into the `BulkDocs` class instead. We can change the constructor of `BulkDocs` to become something like this instead:

```php
static public function createInstance(ContainerInterface $container, WorkspaceManagerInterface $workspace_manager, WorkspaceInterface $workspace) {
  return new static(
    $workspace_manager,
    $workspace,
    $container->get('relaxed.stub_entity_processor')
  );
}
```